README.md

```diff
@@ -19,7 +19,134 @@ HeFQUIN is a query federation engine for heterogeneous federations of graph data
 ### Current Limitations
 * HeFQUIN does not yet have a source selection component. All subpatterns of the queries given to HeFQUIN need to be wrapped in SERVICE clauses.
 
-### Publications related to HeFQUIN
+## Quick Start
+
+HeFQUIN can be used as a Web service or via a Command Line Interface (CLI) tool. 
```

Comment:
Third option: as a Java library linked directly into Java applications (perhaps also for other JVM-based languages such as Scala?)